source/blog/2018-12-14-the-ember-times-issue-77.md

```diff
@@ -22,8 +22,13 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Components patterns in Ember Blog 📚🧐](https://twitter.com/TheMacsour/status/1072088269742112768)
+
+[@Exelord](https://github.com/Exelord) published a blog post on [component patterns in Ember](https://medium.com/macsour/components-patterns-in-ember-js-5e6fc6eea28f). In the article, [@Exelord](https://github.com/Exelord) gathered **component patterns** around the Ember community for readers to better understand what kind of component pattern would best match your particular use case.
```

Comment:
I think since `Exelord` is mentioned in each sentence, this might also be written as follows for more variety:

```suggestion
[@Exelord](https://github.com/Exelord) published a blog post on [component patterns in Ember](https://medium.com/macsour/components-patterns-in-ember-js-5e6fc6eea28f). The article presents a collection of **component patterns** used in the Ember community for readers to better understand what kind of component pattern would best match their particular use case.
```

Also updated `your particular use case` -> `their particular use case`